sdk/nodejs/tests/runtime/langhost/cases/051.resource_creation_in_function_with_result/index.js

```diff
@@ -0,0 +1,12 @@
+module.exports = () => {
```

Comment:
Can we add a variant of this where the resource is created in an async function?

Reply:
sure